src/App.jsx

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
 import './App.scss';
+import { Sum } from './components/Sum';
 
 const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
+    <Sum a={2} and b={3} />
+    <Sum a={-5} and b={5} />
```

Comment:
veery nice 👍 

but why passing `and` to props?
```suggestion
    <Sum a={2} b={3} />
    <Sum a={-5} b={5} />
```